test/controllers/CheckYourAppealController.test.ts

```diff
@@ -91,29 +90,163 @@ describe('CheckYourAppealController', () => {
+
+                emailService.received().send({
+                    to: 'appeals.ch.fake+DEFAULT@gmail.com',
+                    subject: 'Appeal submitted - 00345567',
+                    body: {
+                        templateName: 'lfp-appeal-submission-internal',
+                        templateData: {
+                            companyNumber: '00345567',
+                            userProfile: {
+                                email: 'test'
+                            },
+                            reasons: {
+                                other: {
+                                    title: 'I have reasons',
+                                    description: 'they are legit'
+                                }
+                            }
+                        }
+                    }
+                });
             });
 
-            const app = createApp(session, container => {
-                container.rebind(EmailService).toConstantValue(emailService);
+            it('should send internal email to SC appeals team', async () => {
+                const emailServiceSC = createSubstituteOf<EmailService>(service => {
+                    service.send(Arg.any()).returns(Promise.resolve());
+                });
+
+                const app = createApp(sessionSC, container => {
+                    container.rebind(EmailService).toConstantValue(emailServiceSC);
+                });
+
+                await request(app).post(CHECK_YOUR_APPEAL_PAGE_URI);
+
+                emailServiceSC.received().send({
+                    to: 'appeals.ch.fake+SC@gmail.com',
+                    subject: 'Appeal submitted - SC345567',
+                    body: {
+                        templateName: 'lfp-appeal-submission-internal',
+                        templateData: {
+                            companyNumber: 'SC345567',
+                            userProfile: {
+                                email: 'test'
+                            },
+                            reasons: {
+                                other: {
+                                    title: 'I have reasons',
+                                    description: 'they are legit'
+                                }
+                            }
+                        }
+                    }
+                });
             });
 
-            await request(app).post(CHECK_YOUR_APPEAL_PAGE_URI);
-
-            emailService.received().send({
-                to: 'test',
-                subject: 'Confirmation of your appeal - 00345567 - Companies House',
-                body: {
-                    templateName: 'lfp-appeal-submission-confirmation',
-                    templateData: {
-                        companyNumber: '00345567',
-                        userProfile: {
-                            email: 'test'
+            it('should send internal email to NI appeals team', async () => {
```

Comment:
I am not sure we need all these 3 test cases. When you unit tested service that make decision who receives email I do not think there is a need to simulate all three scenarios here. You can just mock behaviour of the service and have one integration test case.